app/serializers/revenue_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class RevenueSerializer
```

Comment:
Revenue poro is important because now our serializers look very standard and we can send in id and revenue as attributes to be rendered.